src/client/Client.js

```diff
@@ -477,8 +479,7 @@ export default class Client {
                     });
             }
             if (Platform.OS === 'ios') {
```

Comment:
Looks like the same code for both platforms

Reply:
the same for `callConference` API

Reply:
done